pkg/reconcile/pipeline/handler/collect/impl_test.go

```diff
@@ -1030,6 +1030,71 @@ var _ = Describe("Integration Collect definitions + items", func() {
 
 })
 
+var _ = Describe("Handle invalid sliceOfMap annotations", func() {
+	var (
+		mockCtrl *gomock.Controller
+	)
+
+	BeforeEach(func() {
+		mockCtrl = gomock.NewController(GinkgoT())
+	})
+
+	AfterEach(func() {
+		mockCtrl.Finish()
+	})
+
+	type testCase struct {
+		serviceContent map[string]interface{}
+	}
+
+	It("should set an error condition",
```

Comment:
this should be actually added around this line https://github.com/redhat-developer/service-binding-operator/blob/master/pkg/reconcile/pipeline/handler/collect/impl_test.go#L390

The test name should describe the fixture and the expected outcome - "should set an error condition" does not say anything about under what condition and kind of error.